react.component/src/form/Switcher/Switcher.tsx

```diff
@@ -0,0 +1,14 @@
+import { forwardRef } from 'react';
+import './switcher.scss';
+
+type SwitcherProps = {
+  title: string;
+  id: string;
+};
+export const Switcher = forwardRef<HTMLInputElement, SwitcherProps>((props, ref) => (
+  <div className="switcher-wrapper">
+    {props.title}
+    <input type="checkbox" id={props.id} className="switcher-input" ref={ref} />
+    <label htmlFor={props.id} className="switcher-label"></label>
```

Comment:
probably titile should be wrapped in the label

Reply:
if putting title into label tag stytes of castom switcher break :(